app/src/main/java/org/kiwix/kiwixmobile/KiwixWebViewClient.java

```diff
@@ -89,6 +90,14 @@ public void onPageFinished(WebView view, String url) {
             );
         view.addView(help);
         TextView contact = help.findViewById(R.id.welcome21);
+        FloatingActionButton share = help.findViewById(R.id.share);
+        share.setOnClickListener(shareview-> {
+          Intent sharingIntent = new Intent(Intent.ACTION_SEND);
+          sharingIntent.setType("text/plain");
+          sharingIntent.putExtra(Intent.EXTRA_SUBJECT , shareview.getContext().getString(R.string.sharing_message) );
+          sharingIntent.putExtra(Intent.EXTRA_TEXT , shareview.getContext().getString(R.string.sharing_body));
+          shareview.getContext().startActivity(Intent.createChooser( sharingIntent , "Share via"));
```

Comment:
Not sure what this string is doing but it should either be in strings.xml or in Constants.java